sdk/keyvault/keyvault-certificates/tests/CRUD.test.ts

```diff
@@ -189,4 +190,99 @@ describe("Certificates client - create, read, update and delete operations", ()
       "Unexpected certificate name in result from getKey()."
     );
   });
+
+  it("can create, read, and delete a certificate's issuer", async function() {
+    const certificateName = testClient.formatName(`${prefix}-${this!.test!.title}-${suffix}`);
+    await client.createCertificate(certificateName, basicCertificateProperties);
+    const issuerName = "issuerName";
+
+    // Create
+    await client.setCertificateIssuer(certificateName, "Test", {
+      name: issuerName
+    });
+
+    let getResponse: any;
+
+    // Read
+    getResponse = await client.getCertificateIssuer(certificateName);
```

Comment:
This doesn't seem quite right. if I'm remembering right, the certificate issuer is for the keyvault not for the certificate.  Maybe @schaabs can confirm.

One example for the docs: https://docs.microsoft.com/en-us/rest/api/keyvault/setcertificateissuer/setcertificateissuer

Reply:
What I mean is that getCertificateIssuer doesn't take a certificate name:

```
  public async getCertificateIssuer(issuerName: string, options?: RequestOptions): Promise<CertificateIssuer> {
    let result = await this.client.getCertificateIssuer(this.vaultBaseUrl, issuerName, options);

    return result._response.parsedBody;
  }
```

Reply:
ohh I understand your point. I'll fix it

Reply:
Resolved!